src/parse.ts

```diff
@@ -25,7 +25,12 @@ import { traversableToJSON } from './traversableToJSON';
  *
  * @returns {object}
  */
```

Comment:
All the documentation of the options should migrate to the 'ParseOptions' interface.

The @default value should also be specified.

Reply:
i added default value 
could you please check?